frontend/public/components/_quota.scss

```diff
@@ -1,12 +1,9 @@
 .co-resource-quota-chart-row {
   display: flex;
   flex-wrap: wrap;
-  justify-content: center;
-  margin-bottom: 20px;
-}
-
-.co-resource-quota-chart-row--compact {
-  margin-bottom: 2px;
+  justify-content: space-evenly;
+  margin-left: 40px;
```

Comment:
I'd be inclined to drop both these margin additions as they result in premature wrapping that's not necessarily desirable.

Reply:
fixed